.github/workflows/android.yml

```diff
@@ -0,0 +1,112 @@
+name: android
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  push:
+    branches:
+      - master
+
+jobs:
+  ci:
+    if: github.event.pull_request.draft == false
+
+    strategy:
+      matrix:
+        arch: [arm64-v8a, x86_64]
+
+    runs-on: ubuntu-latest
+    container: ghcr.io/f3d-app/f3d-ci
+
+    env:
+      VTK_SHA: 35ff12bbc080528fb0dc3f43ac3288e10ce22c85
+      NDK_VERSION: r21e
+      API_LEVEL: 27
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        path: 'source'
+        fetch-depth: 0
+        lfs: 'false'
+
+    - name: Cache NDK
+      id: cache-ndk
+      uses: actions/cache@v2
+      with:
+        path: android-ndk-${{env.NDK_VERSION}}
+        key: android-ndk-${{env.NDK_VERSION}}-2
+
+    - name: Install NDK
+      if: steps.cache-ndk.outputs.cache-hit != 'true'
+      working-directory: ${{github.workspace}}
+      run: |
+        apt-get install unzip
+        wget https://dl.google.com/android/repository/android-ndk-${{env.NDK_VERSION}}-linux-x86_64.zip
+        unzip android-ndk-${{env.NDK_VERSION}}-linux-x86_64.zip
+
+    - name: Dependencies Dir
+      working-directory: ${{github.workspace}}
+      run: mkdir dependencies
+
+    - name: Cache VTK
+      id: cache-vtk
+      uses: actions/cache@v2
+      with:
+        path: dependencies/vtk_build
+        key: vtk-${{env.VTK_SHA}}-2-android-${{env.API_LEVEL}}-${{matrix.arch}}
+
+    - name: Checkout VTK
```

Comment:
you could improve the VTK action to support what you need

Reply:
Create a custom action then :)

Reply:
eg: `./source/.github/actions/vtk-android-install-dep`. And move vtk_commit_sha one directory up since it will be used by both actions